web/src/lib/components/asset-viewer/video-viewer.svelte

```diff
@@ -48,21 +25,19 @@
 	transition:fade={{ duration: 150 }}
 	class="flex place-items-center place-content-center h-full select-none"
 >
-	{#if asset}
-		<video
-			controls
-			class="h-full object-contain"
-			on:canplay={handleCanPlay}
-			on:ended={() => dispatch('onVideoEnded')}
-		>
-			<source src={videoUrl} type="video/mp4" />
```

Comment:
I remember we ran into issue with the player if I don't include the type property in the source element

Reply:
Shouldn't be an issue as long as the server sends the correct Content-Type (which it does) but I've moved back to using the source element.